lib/Signaling/Listener.php

```diff
@@ -136,6 +136,7 @@ protected function notifyRoomModified(ARoomModifiedEvent $event): void {
 			ARoomModifiedEvent::PROPERTY_READ_ONLY,
 			ARoomModifiedEvent::PROPERTY_SIP_ENABLED,
 			ARoomModifiedEvent::PROPERTY_TYPE,
+			ARoomModifiedEvent::PROPERTY_MENTION_PERMISSIOINS,
```

Comment:
We don't need to notify about this, it does not impact calls or require users to reload the page/call